src/client/pydaos/pydaos_shim.c

```diff
@@ -1017,18 +1016,11 @@ static struct PyModuleDef moduledef = {
 };
 
 PyMODINIT_FUNC PyInit_pydaos_shim_3(void)
-#else
-void
-initpydaos_shim_27(void)
-#endif
+
```

Comment:
Should probably remove this newline. I'm not sure why the checkpatch stage didn't check this as it normally would complain about a new line after a function declaration.